Error : when validating test1::main
 --> test_resources/commit_3dc35/test1.yr:(12,4)
12  ┃ fn main () {
    ╋    ^^^^
    ┃ Error : test1::foo called with {1u64} works with multiple candidates
    ┃  --> test_resources/commit_3dc35/test1.yr:(15,9)
    ┃ 15  ┃     foo (1u64);
    ┃     ╋         ^
    ┃     ┃ Note : candidate test1::foo (a : i32)-> void
    ┃     ┃ Note : candidate test1::foo (a : i64)-> void
    ┃     ┗━━━━━━ 
    ┗━━━━━┻━ 
